### ISSUE TITLE
Incorrect Sourcemaps caused by vike:envVars plugin

### ISSUE THREAD

**Issue by mpiniarski** [vikejs/vike#1481]
### Description

After migrating to `vike` from `vite-plugin-ssr` there are errors during source map generation:

```
(vike:envVars plugin) Sourcemap is likely to be incorrect: a plugin (vike:envVars) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
```

and source maps are indeed incorrect.

**Comment by brillout** [comment]
Thanks for opening the bug report. I'll fix it later today.

**Comment by brillout** [comment]
Fix pre-released in `0.4.161-commit-0e9c635`.

**Comment by mpiniarski** [comment]
I can confirm the fix works.
